doc/user-guide.adoc

```diff
@@ -912,6 +917,107 @@ data:
 type: kubernetes.io/tls
 ----
 
+=== Affinity
+
+==== Node Affinity
+
+===== Basic node affinity
+
+Using `nodeAffinityLabels` field it is possible to set required labels for pod scheduling on specific nodes:
+
+[source,yaml]
+----
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: my-app
+  namespace: test
+spec:
+  applicationImage: quay.io/my-repo/my-app:1.0
+  affinity:
+    nodeAffinityLabels:
+      customNodeLabel: label1, label2
+      customNodeLabel2: label3
+----
+
+===== Advanced node affinity example:
+
+Require a large node type and prefer zone A and zone B
+
+[source,yaml]
+----
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: my-app
+  namespace: test
+spec:
+  applicationImage: quay.io/my-repo/my-app:1.0
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key:  node.kubernetes.io/instance-type
+            operator: In
+            values:
+            - large
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 60
+        preference:
+          matchExpressions:
+          - key: failure-domain.beta.kubernetes.io/zone
+            operator: In
+            values:
+            - zoneA
+      - weight: 20
+        preference:
+          matchExpressions:
+          - key: failure-domain.beta.kubernetes.io/zone
+            operator: In
+            values:
+            - zoneB
+----
+
+==== Pod Affinity and Anti-Affinity
+
+Pod affinity and anti-affinity allow you to constrain which nodes your pod is eligible to be scheduled based on labels on pods that are already running on the node rather than based on labels on node.
+
+Bellow example shows required pod affinity where pod for service A and B should be co-located in the same zone. Additionally, it is preferred not to schedule on hosts that are running service C.
```

Comment:
Tip: Avoid using "below", "above", "right", and "left"  to give the user direction for the page that they are reading.. The reason is that these words are accessibility violations because screen readers don't have the ability to give their users a sense of direction.

Are Service A and Service B in the same pod or in separate pods? Here is possible wording. I'm assuming that the two services are in different pods. maybe they aren't.
Change to "The following example shows that pod affinity is required and that the pods for service A and service B must be in the same zone. The preference for hosts that are running on service C is that they are not scheduled.".